src/app/layout.tsx

```diff
@@ -16,6 +16,16 @@ export const metadata: Metadata = {
   title: "宝灯桃汁スロットマシーン2025春",
   description: "宝灯桃汁スロットマシーン2025春",
   metadataBase: new URL("https://anatofuz.net/momojiruslot202504"),
+  openGraph: {
+    images: [
+      {
+        url: "/ogp/og-image.png", // public/ogp/og-image.png を指定
```

Comment:
今回クソ雑にanatofuz.net以下のサブディレクトリ(not サブドメイン)なので相対パスで指定したいンゴね
```suggestion
        url: "./ogp/og-image.png", // public/ogp/og-image.png を指定
```